infrastructure/cdn-in-a-box/enroller/Dockerfile

```diff
@@ -17,18 +17,12 @@
 
 FROM golang:1.14.2 AS enroller-builder
 
-# enroller source and dependencies
-COPY ./lib/ /go/src/github.com/apache/trafficcontrol/lib/
-COPY ./vendor/ /go/src/github.com/apache/trafficcontrol/vendor/
-COPY ./traffic_ops/client/ /go/src/github.com/apache/trafficcontrol/traffic_ops/v2-client/
```

Comment:
I think the intention was to keep the enroller using the v2 client until it could be updated to use the v3 client (with the breaking servers changes). See https://github.com/apache/trafficcontrol/pull/4700/files#diff-1bafc9163f1489d723b8f430867c0a0dR23

Reply:
Gotcha. Changed to v2 client in 8f7763330